### ISSUE TITLE
Uncaught error when retrieving columns info with jsonb[] with default values

### ISSUE THREAD

**Issue by ntma** [knex/knex-schema-inspector#72]
Hi knex-schema-inspector team,

Due to the nature of [this regex](https://github.com/knex/knex-schema-inspector/blob/0f5f9835ef93a076b6f1ee582f300a332d723727/lib/dialects/postgres.ts#L73), it seems that the default value of columns defined as `json[]` or `jsonb[]` will be parsed as Json objects and throw an uncaught error.

### Reproduction

* define a table with the following column;
```sql
   my_column jsonb[] default ARRAY[]::jsonb[],
```
* retrieve information for the columns in our table;

In this case, the parsing will look like `JSON.parse("ARRAY[]")`, which is not a valid format.

**Comment by ntma** [comment]
I am sorry for the confusion @rijkvanzanten .

TLDR we have in one side the parsing that the `pg` module does to retrieve values from the database, which I believe is not related to this issue.

On the other side we have the parsing that `knex-schema-inspector` does to extract information from schema definitions. And if I am not wrong, currently this parsing does not support the PostgreSQL syntax for array's as default values (since there is no direct mapping to JavaScript for `Array[]`).

**Comment by aidenfoxx** [comment]
@rijkvanzanten Isn't this (kinda) fixed with https://github.com/knex/knex-schema-inspector/pull/117 since we now return unparsed JSON strings?

**Comment by rijkvanzanten** [comment]
Yes!
